### PR TITLE
Add lib defs for redux-entitize

### PR DESCRIPTION
**Pull request by leoselig**

### CODE REVIEW

**Comment by GAntoine** [comment]
Hey @leoselig, since `redux-entitize` seems to be your project, would you be interested in adding type definitions directly into it? I could probably find a couple other projects which have done the same thing, without rewriting everything with flow.

**Comment by leoselig** [comment]
@GAntoine
Thank your for your kind suggestion. I could absolutely do that of course.

But I'm a little confused as to what goal flow-typed actually persues. So far, it seems to be the only (and officially recommended) way of providing flow type definitions based both on the package version and the flow version of the dependent project.

I also don't see any outline of a strategy in this repository that declares what packages are "worth" to be managed via flow-typed. I thought it was open for everyone that publishes to the 
registry.

I'd be sincerely eager for the maintainers to clarify that strategy.

**Comment by GAntoine** [comment]
Thanks for your response! To be perfectly honest I don't know of any such strategy, I do think that the typedefs for any package are welcome here.

From what I've gathered, when a user of a package would like flow typings, but the package maintainers don't have them/won't add them, they can use flow-typed instead. 

The opposite case would be if the package itself is either written in flow, or has typings. For example, someone opened a PR to add types for `recompose` on #624, but then recompose [added their own typings](https://github.com/acdlite/recompose/pull/428), so flow-typed was unnecessary.

@calebmer Any thoughts on this?

File: definitions/npm/redux-entitize_v0.2.x/flow_v0.45.x-/redux-entitize_v0.2.x.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { type Schema } from "normalizr";
```
Comment (GAntoine) [inline definitions/npm/redux-entitize_v0.2.x/flow_v0.45.x-/redux-entitize_v0.2.x.js]: This won't work as you'd expect it to...

**Comment by leoselig** [comment]
I'm afraid I don't have time for this PR at the moment. Will probably get back to it in 1 or 2 weeks.
Thanks for your feedback so far and the discussion on the distribution of type defs.

If you want to tidy up the PR list, feel free to close it. Then I'll open up another one one I'm focused enough.

File: definitions/npm/redux-entitize_v0.2.x/flow_v0.45.x-/redux-entitize_v0.2.x.js (hunk 1 above)
Comment (leoselig) [reply definitions/npm/redux-entitize_v0.2.x/flow_v0.45.x-/redux-entitize_v0.2.x.js]: I realized, should have activated the linter to reveal the pointless import

**Comment by GAntoine** [comment]
Thanks for the heads up, I'm happy to keep this one open since you're still around. Ping me when you're back to working on it and we'll get it merged lickity split.

**Comment by GAntoine** [comment]
Updated and merged PR, thanks for the 💪 !

**Pull request merged**